File: app/src/main/java/com/mohammedsazid/android/aiub/LoginActivity.java
package com.mohammedsazid.android.aiub;

import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.preference.PreferenceManager;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.AppCompatEditText;
import android.view.View;
import android.widget.Button;

public class LoginActivity extends AppCompatActivity implements View.OnClickListener {

    private AppCompatEditText username;
    private AppCompatEditText password;
    private Button saveBtn;

    private SharedPreferences prefs;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        prefs = PreferenceManager.getDefaultSharedPreferences(this);
        bindViews();

        saveBtn.setOnClickListener(this);
    }

    private void bindViews() {
        username = (AppCompatEditText) findViewById(R.id.username);
        password = (AppCompatEditText) findViewById(R.id.password);
        saveBtn = (Button) findViewById(R.id.save);
    }

    @Override
    public void onClick(View v) {
        String usernameStr = username.getText().toString();
        String passwordStr = password.getText().toString();

        SharedPreferences.Editor editor = prefs.edit();
        editor.putString(getString(R.string.pref_username_key), usernameStr);
        editor.putString(getString(R.string.pref_password_key), passwordStr);
        editor.apply();

        startActivity(new Intent(this, MainActivity.class));
        finish();
    }
}

File: app/src/main/java/com/mohammedsazid/android/aiub/LoginActivity.kt
package com.mohammedsazid.android.aiub

import android.annotation.SuppressLint
import android.content.Intent
import android.content.SharedPreferences
import android.graphics.Color
import android.os.Bundle
import android.preference.PreferenceManager
import androidx.appcompat.app.AppCompatActivity
import android.view.View

import kotlinx.android.synthetic.main.activity_login.*

class LoginActivity : AppCompatActivity(), View.OnClickListener {

    private var prefs: SharedPreferences? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        window.decorView.setBackgroundColor(Color.WHITE)

        prefs = PreferenceManager.getDefaultSharedPreferences(this)

        saveBtn.setOnClickListener(this)
    }

    @SuppressLint("ApplySharedPref")
    override fun onClick(v: View) {
        val usernameStr = username.text.toString()
        val passwordStr = password.text.toString()

        prefs?.edit()
                ?.putString(getString(R.string.pref_username_key), usernameStr)
                ?.putString(getString(R.string.pref_password_key), passwordStr)
                ?.commit()

        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}
